### DISCUSSION TITLE
Improve draggability of data model view in Settings

### DISCUSSION THREAD

**Discussion by TomS-** [directus/directus#9125] [Feature Requests]
The new way of laying out and organising collections into folders and allowing collections to sit under collections is a great addition to Directus. It is however, difficult to drag the collection where you want it to be.

https://user-images.githubusercontent.com/9141017/138725551-8fe99a21-216a-4735-9361-9838186bb038.mp4

As a side-note: "Collection Navigation" still exists under user roles, but have no effect. I assume that will likely be removed in favor of this new way of laying out collections, but what would be nice is to have a "expanded", "collapsed" options like previously.

Edit: grammar.

**Comment by rijkvanzanten** [comment]
The hitboxes are very finicky to get right using SortableJS. 

@Nitwel is this something you'd like to experiment with?

**Reply by benhaynes** [reply]
I wonder if we can "fix" this with some new hit zones (maybe with pseudoelements)? Below are some sketches around how the hit zones might be able to add "fake" drag to the right to indent...

![Screen Shot 2021-10-25 at 11 45 23 AM](https://user-images.githubusercontent.com/522079/138728210-58e70067-541c-4ad0-b87f-2fe8e94bcdea.png)

Not sure what we have now though.

**Reply by rijkvanzanten** [reply]
Also wondering how much more we can squeeze out of this before we hit the ceiling of what we can do with vue-draggable 🤔  It can be particularly pesky when it comes to things like this

**Reply by Nitwel** [reply]
I could look into how vuedraggable calculates its zones / hitboxes and enlarge those maybe. Have to see when I find time for that but I'll keep it in my head.

**Reply by rijkvanzanten** [reply]
Yeah, no pressure! Seeing that both you and I have now dealt with vue-draggable quite a bit, I was thinking you might have some smart ideas to get this working ideally 🙂

**Reply by TomS-** [reply]
Would the UI for field groups not work for this? All be it slightly tweaked. If I recall this puts a visual "dropzone" in place which may even be favourable to larger hit boxes as it gives the user something visual they can be confident in.

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
